composer.json

```diff
@@ -15,5 +17,12 @@
             "type": "composer",
             "url": "https://packages.drupal.org/8"
         }
+    },
+    "extra": {
+        "patches": {
+            "drupal/focal_point": {
+                "Preview modal does not load js/css - https://www.drupal.org/project/focal_point/issues/3126288": "https://www.drupal.org/files/issues/2020-04-09/3126288-2.patch"
```

Comment:
Hey @MdNadimHossain can I suggest you help review this on drupal.org please. It will help it get into the mainline code. You just have to pipe in with a me too and that we're using the patch in real life.

Reply:
ok will do that.